src/remote.ts

```diff
@@ -38,18 +40,13 @@ export class Remote {
     // If the URI passed doesn't have the proper prefix ignore it. We don't need
     // to do anything special, because this isn't trying to open a Coder
     // workspace.
-    if (!authorityParts[1].startsWith(Remote.Prefix)) {
-      return
-    }
+    // if (!authorityParts[1].startsWith(Remote.Prefix)) {
+    //   return
+    // }
```

Comment:
This needs to be checked, otherwise Coder will activate on any remote SSH connection, which will cause errors to appear on non-Coder connections.